### ISSUE TITLE
[Documentation] Remove link to register for outdated event on fastboot docs page

### ISSUE THREAD

**Issue by akankshadharkar** [ember-fastboot/fastboot-website#117]
We should remove the link to register for an event ( Ember Fastboot SSR & Beyond) that ended 2 weeks days back which is showing on the page
It's a good beginner issue but I can take it up if required

**Comment by suchitadoshi1987** [comment]
@akankshadharkar you are absolutely right. PR is super welcome :) Thanks for volunteering for this one!
